contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtTripsAnalyser.java

```diff
@@ -385,7 +385,7 @@ public static void writeVehicleDistances(Map<Id<Vehicle>, DrtVehicleDistanceStat
 						+ format.format(vehicleState.totalDistance) + ";"//
 						+ format.format(vehicleState.totalOccupiedDistance) + ";"//
 						+ format.format(vehicleState.totalDistanceByOccupancy[0]) + ";"//
-						+ format.format(vehicleState.totalRevenueDistance));
+						+ format.format(vehicleState.totalPassengerMeter));
```

Comment:
`totalPassengerDistance`? So that it follows the column names ("totalPassengerDistance" and "averagePassengerDistance")

Reply:
or `totalPassengerTraveledDistance`?